docs-ref-services/mysql.md

```diff
@@ -23,7 +23,7 @@ Learn more about [Azure Database for MySQL](https://docs.microsoft.com/azure/MyS
 
 ## Client Package
 
-### Install npm module
+### Install the npm module
 
 Use npm to install the MySQL client module.
```

Comment:
This says to install the `mysql` module, but the over view links to a different module which has documentation to install with a different module name: https://github.com/sidorares/node-mysql2#installation. Which one should it be?